README.md

```diff
@@ -1 +1,27 @@
-# To-do-list
\ No newline at end of file
+# To-do-list
+A to-do list is a app that allow the user to organize the user's day and allow him to add a task, to remove it, to edit it and then to mark it as complet if it is
+
+# Build with
+
+HTML, CSS, Javascript, Webpack
+
+# Live Demo
+
+[link to my to-do list]()
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+```
+ git clone 
+ https://github.com/To-do-list/
+ 
+
+```
```

Comment:
- [ ] [OPTIONAL] Kindly ensure you add sets of instructions on how to get a copy of the project locally and how to run the project for visitors to be aware. I checked your `package.json` file to run the project.